### ISSUE TITLE
Aux instead of cop ? 

### ISSUE THREAD

**Issue by ftyers** [UniversalDependencies/UD_Latvian-LVTB#1]
Should `aux` here be `cop` ? 

```
# a-DIENA_intervija_20130128-p28s1
1       Tā      tā      PRON    pd0fsnn Case=Nom|Gender=Fem|Number=Sing|PronType=Dem    4       nsubj   _       _
2       ir      būt     VERB    vcnipii30an     Mood=Ind|Negative=Neg|Person=3|Tense=Pres|VerbForm=Fin|Voice=Act        4       aux     _       _
3       tā      tā      DET     pd0fsnn Case=Nom|Gender=Fem|Number=Sing|PronType=Dem    4       det     _       _
4       problēma        problēma        NOUN    ncfsn4  Case=Nom|Gender=Fem|Number=Sing 0       root    _       SpaceAfter=No
5       .       .       PUNCT   zs      _       4       punct   _       _
```

**Comment by lauma** [comment]
Yes, it should. I will check the conversion procedure (this seems rather straight-forward case at the first glance) and upload corrected data.

**Comment by lauma** [comment]
Should be fixed in 0317bd7. If it is not the case, feel free to reopen the issue.
